app/controllers/api/v1/users/squads_controller.rb

```diff
@@ -12,7 +16,12 @@ def create
   end
 
   def destroy
-    user_squad = UserSquad.find(params[:id])
-    user_squad.destroy
+    user_squad = UserSquad.find_by(id: params[:id])
+    if !user_squad.nil?
+      user_squad.destroy
+      render json: { description: "UserSquad deleted successfully" }, status: 204 
```

Comment:
On `destroy` it is very common to just return

```
render json: {}, status: :ok
```

If you're already consuming the message on the FE then just consider this a suggestion.